build/config/android/config.gni

```diff
@@ -16,7 +16,7 @@ if (is_android) {
 
   if (!defined(default_android_sdk_root)) {
     default_android_sdk_root = "//third_party/android_tools/sdk"
-    default_android_sdk_version = "28"
+    default_android_sdk_version = "29"
     default_android_sdk_build_tools_version = "28.0.3"
```

Comment:
Shouldn't we update the build tools as well?

Reply:
I'm not sure which version of the build tools go with this Android SDK. Is there some resource we can reference for this? I pulled the latest version of the build tools from Android Studio after installing SDK version 29

Reply:
Looks like 29.0.2 is available now.